packages/react-devtools-extensions/src/background.js

```diff
@@ -2,17 +2,41 @@
 
 'use strict';
 
+import {IS_FIREFOX} from './utils';
+
 const ports = {};
 
-const IS_FIREFOX = navigator.userAgent.indexOf('Firefox') >= 0;
+if (!IS_FIREFOX) {
+  // Manifest V3 method of injecting content scripts (not yet supported in Firefox)
+  // Note: the "world" option in registerContentScripts is only available in Chrome v102+
```

Comment:
Should we have a fallback for Chrome < v102?. Ex. maybe it's okay for older Chrome versions to have a race condition or block when using React DevTools (with a warning to install the newer version)

Reply:
Yeah good point. Let me try find a good place to do that